docs/Updating-gene-and-gene_alias-tables.md

```diff
@@ -36,20 +36,20 @@ TRUNCATE TABLE gene;
 
 4- Restart cBioPortal (restart webserver) to clean-up any cached gene lists.
 
-5- You probably also want to update the gene lengths. This is dependent on the reference genome you want to use. Download .annotation.gtf.gz from the latest GENCODE release from http://www.gencodegenes.org/releases/current.html for hg38 / GRCh38. For the hg19 / GRCh37, click on 'Go to GRCh37 version of this release'.
+5- You probably also want to update the gene lengths. To do so, download .annotation.gtf.gz of the [latest GENCODE release for hg19 / GRCh37](http://www.gencodegenes.org/releases/26lift37.html) for human and [mm10 / GRCm38](http://www.gencodegenes.org/mouse_releases/current.html) for mouse.
 
 After downloading, go to your downloads directory, decompress the file and add it as an argument (--gtf) in the next step.
 
 6- To import gene data type the following commands when in the folder `<your_cbioportal_dir>/core/src/main/scripts`:
 
 ```
  export PORTAL_HOME=<your_cbioportal_dir>
-./importGenes.pl --genes <Homo_sapiens_gene_info.txt> --gtf <gencode.v25.annotation.gtf>
+./importGenes.pl --genes <ncbi_gene_info.txt> --gtf <gencode.v25.annotation.gtf>
```

Comment:
No. In line 5 specifies which `gene_info.txt` file from NCBI you should use if you have a human instance or a mouse one.